README.md

```diff
@@ -40,11 +42,51 @@ The Battery Toolkit “Commands“ menu and its menu bar extra allow you to issu
 |:--:| 
 | *Battery Toolkit Menu Bar Extra* |
 
+# Install
+
+### Manual Install
+1. Go to the [releases](https://github.com/mhaeuser/Battery-Toolkit/releases/latest)
+2. Download the newest non-dSYM build (e.g. `Battery-Toolkit-1.6.zip`)
+3. Unzip it (double click)
+4. Drag it into your Applications folder
+5. Right click the `Battery Toolkit.app`, then click "Open", then allow in the dialog box
+6. If that doesn't work, go to `System Settings -> Privacy & Security` and you should see a prompt at the bottom of the screen allow opening Battery Toolkit. Allow it.
+7. Open Battery Toolkit again from Applications folder.
+
+### Install via Homebrew
+1. Install [Homebrew](https://brew.sh) if you haven't already.
+2. Open Terminal and run `brew tap mhaeuser/mhaeuser`
+2. Then run `brew install battery-toolkit --no-quarantine`
+4. Type password if prompted.
+5. Open Battery Toolkit from your Applications folder
+
+> [!CAUTION]
+> To ensure there is no chance of interference, please turn “Optimized Charging” **off** when Battery Toolkit is in use.
+
+# Usage
+1. Open Battery Toolkit
+2. Click the menu bar icon
+3. Configure your desired setting through the menu options
+
+If you prefer, you can quit the GUI through the menu bar extra. Battery Toolkit will keep running in the background.
+If you want to change any settings, simply re-open the app.
+
+# Uninstall
+
+1. Enable "Pause Background Activity" from the Battery Toolkit menu bar item
+2. Move the app to the trash.
+
 # Compatibility
 
-Battery Toolkit currently supports only Apple Silicon Macs.
+* **Battery Toolkit currently only supports Apple Silicon Macs.**
+* M4 series MacBooks may have some small issues [#55](https://github.com/mhaeuser/Battery-Toolkit/issues/55)
+
+# Known Issues / Caveats
+
+* Battery Toolkit disables sleep while it is charging, because it has to actively disable charging once reaching the maximum. [#83](https://github.com/mhaeuser/Battery-Toolkit/issues/83)
+* Battery Toolkit cannot control the charge state when the machine is shut down. If charger remains plugged in while the Mac is off, battery will charge to 100% as normal.
```

Comment:
This is not a caveat of Battery Toolkit, but maybe we should add that sleep should usually be disabled when the power adapter is disabled, as this will exit Clamshell mode and the machine will sleep immediately if the lid is closed.